custom/enikshay/integrations/ninetyninedots/repeater_generators.py

```diff
@@ -92,8 +93,16 @@ def create(cls, person_case, episode_case):
         )
 
 
+class NinetyNineDotsBasePayloadGenerator(BasePayloadGenerator):
+    def handle_exception(self, exception, repeat_record):
+        if isinstance(exception, RequestConnectionError):
+            update_case(repeat_record.domain, repeat_record.payload_id, {
+                "dots_99_error": u"RequestConnectionError: {}".format(unicode(exception))
+            })
```

Comment:
We handle the exception in the repeater code, but we write back to the case that this error occurred when this case was being submitted. 
It would be great if we had a way of pulling a report that showed you each payload that failed and the reason why. I didn't have time to build that when making the repeaters the first time around, so this is our hacky way of logging it.